mia/medium/famicom.cpp

```diff
@@ -668,10 +703,10 @@ auto Famicom::analyzeINES(vector<u8>& data) -> string {
     s += "      volatile\n";
   }
 
-  if(eeprom) {
+  if(prgnvram) {
     s += "    memory\n";
     s += "      type: EEPROM\n";
-    s +={"      size: 0x", hex(eeprom), "\n"};
+    s +={"      size: 0x", hex(prgnvram), "\n"};
     s += "      content: Save\n";
```

Comment:
I think this is a mistake. prgnvram is reset to 0 above, so I would expect eeprom to be referenced here. Please do try a game from each of the eeprom mappers, if you can. If not, we can ask a tester to take a look.

Reply:
It is a mistake, sorry. I typically rely on test ROMs, but the EEPROM mappers don't have much in this way; I'd appreciate a tester taking a more extensive look as described above.